webrender/src/frame_builder.rs

```diff
@@ -46,12 +46,21 @@ fn make_polygon(
     stacking_context: &StackingContext,
     node: &ClipScrollNode,
     anchor: usize,
+    device_pixel_ratio: f32,
 ) -> Polygon<f32, WorldPixel> {
-    //TODO: only work with `isolated_items_bounds.size` worth of space
```

Comment:
why is this comment removed?

Reply:
I removed this accidentally. I'll add it back.